docs/quickstart.rst

```diff
@@ -21,66 +22,89 @@ Installation
+the custom handler to save records to the database.
 
-        from log_outgoing_requests.formatters import HttpFormatter
+.. code-block:: python
+    :linenos:
+    :emphasize-lines: 1, 8, 14, 20, 25, 26
 
+    from log_outgoing_requests.formatters import HttpFormatter
 
-        LOGGING = {
+
+    LOGGING = {
+        #...,
+        "formatters": {
+            #...,
+            "outgoing_requests": {"()": HttpFormatter},
+        },
+        "handlers": {
             #...,
-            "formatters": {
-                #...,
-                "outgoing_requests": {"()": HttpFormatter},
+            "log_outgoing_requests": {
+                "level": "DEBUG",
+                "formatter": "outgoing_requests",
+                "class": "logging.StreamHandler",  # to write to stdout
             },
-            "handlers": {
-                #...,
-                "log_outgoing_requests": {
-                    "level": "DEBUG",
-                    "formatter": "outgoing_requests",
-                    "class": "logging.StreamHandler",
-                },
-                "save_outgoing_requests": {
-                    "level": "DEBUG",
-                    "class": "log_outgoing_requests.handlers.DatabaseOutgoingRequestsHandler",
-                },
+            "save_outgoing_requests": {
+                "level": "DEBUG",
+                # enabling saving to database
+                "class": "log_outgoing_requests.handlers.DatabaseOutgoingRequestsHandler",
             },
-            "loggers": {
-                #...,
-                "requests": {
-                    "handlers": ["log_outgoing_requests", "save_outgoing_requests"],
-                    "level": "DEBUG",
-                    "propagate": True,
-                },
+        },
+        "loggers": {
+            #...,
+            "requests": {  # the logger name must be 'requests'
+                "handlers": ["log_outgoing_requests", "save_outgoing_requests"],
+                "level": "DEBUG",
+                "propagate": True,
             },
-        }
+        },
+    }
 
-        LOG_OUTGOING_REQUESTS_DB_SAVE = True # save logs enabled/disabled based on the boolean value
-        LOG_OUTGOING_REQUESTS_DB_SAVE_BODY = True # save request/response body
-        LOG_OUTGOING_REQUESTS_EMIT_BODY = True # log request/response body
-        LOG_OUTGOING_REQUESTS_CONTENT_TYPES = [
-                "text/*",
-                "application/json",
-                "application/xml",
-                "application/soap+xml",
-        ] # save request/response bodies with matching content type
-        LOG_OUTGOING_REQUESTS_MAX_CONTENT_LENGTH = 524_288  # maximal size (in bytes) for the request/response body
-        LOG_OUTGOING_REQUESTS_LOG_BODY_TO_STDOUT = True
 
+The library ships with safe defaults for settings - essentially only emitting
+meta-information about requests and responses. To view request and response bodies,
```

Comment:
Perhaps mention where the settings are stored ("...ships with sane defaults for settings, accessible via conf.py - essentially...")?

Reply:
I don't think this is needed, as how we consume the settings in the library is our implementation detail. Users of the library only need to know what they can override.